src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -142,6 +142,14 @@ protected int getNumOfElementOnPage(By cssSelectorBy) {
     return numElementOnPage;
   }
 
+  public void waitSafely(Runnable o) {
+    try {
+      o.run();
+    } catch (TimeoutException e) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)